datanode/entities/aggregated_ledgerentries.go

```diff
@@ -41,23 +45,41 @@ func (ledgerEntries *AggregatedLedgerEntries) ToProto() *v2.AggregatedLedgerEntr
 		}
 	}
 
-	if ledgerEntries.MarketID != nil {
-		marketIDString := ledgerEntries.MarketID.String()
+	if ledgerEntries.AccountFromPartyID != nil {
+		partyIDString := ledgerEntries.AccountFromPartyID.String()
+		if partyIDString != "" {
+			lep.PartyId = &partyIDString
+		}
+	}
+
+	if ledgerEntries.AccountToPartyID != nil {
+		partyIDString := ledgerEntries.AccountToPartyID.String()
+		if partyIDString != "" {
+			lep.PartyId = &partyIDString
+		}
+	}
```

Comment:
is it intended that the toParty overrides the fromParty here? I might have gotten the wrong end of the stick but I kind of expected a proto change to add FromPartyId, ToPartyID etc..

Reply:
No, I forgot about that.
Sorry, will add it now

Reply:
@pscott31 , addressed this:
Removed groupoptions from protos and gqlschema, as well as added the new fields:
https://github.com/vegaprotocol/vega/pull/6834/files#diff-7f6c5ea53fa01231a8a36ffa313993450073ba666c91f11fe0322f33a39b1b86
https://github.com/vegaprotocol/vega/pull/6834/files#diff-be016f833f94c6069e96beb4873414c55bdfb7af1924c9c1b90eb709304ff90e

Reply:
Also corrected the serialization: https://github.com/vegaprotocol/vega/pull/6834/files#diff-1b82afd4970ab61dfe18dca2347cc372dee6b86434f92c37d975daee780796afL27